### ISSUE TITLE
[ldd-ctli] It is not possible to use the CTLI namespace to describe multi-mode instruments

### ISSUE THREAD

**Issue by acraugh** [pds-data-dictionaries/PDS4-LDD-Issue-Repo#181]
**Issue Type**
Multi-mode instruments are not uncommon; I'm assuming it is a bug that they are not supported. If the intention was _not_  to support multi-mode instruments, then this is an enhancement, not a bug.

**Describe the issue identified (if applicable)**
I attempted to use the CTLI dictionary to create a context object to describe the IMACS instrument mounted on the Magellan Baade telescope at Las Campanas Observatory. A summary of instrument specs is here: 

[http://www.lco.cl/wp-content/uploads/2021/03/IMACShandout2021.pdf](http://www.lco.cl/wp-content/uploads/2021/03/IMACShandout2021.pdf)

This instrument acts as both an imager and a spectrograph (among other things). There is no <type> value exactly  corresponding to "spectrograph", and it is in fact a wide-field imager, which is an important characteristic to note. When I encountered the <Type_List_Area> in the <Instrument> object, I went looking for documentation on how to use the CTLI dictionary, but found none (I checked the Standards, the DPH, and the CTLI repo). When I attempted to guess my way through I was informed by validation that all these variations are not schematically valid because of their structure (the attribute values themselves are valid):

Method 1:
```
<Type_List_Area>
  <ctli:Type_List>
    <ctli:type>Imager</ctli:type>
    <ctli:subtype>wide-field</ctli:subtype>
  </ctli:Type_List>
  <ctli:Type_List>
    <ctli:type>Spectrometer</ctli:type>
    <ctli:subtype>spectrograph</ctli:subtype>   
    <ctli:subtype>multi-object</ctli:subtype>
  </ctli:Type_List>
</Type_List_Area>
```
Method 2:
```
<Type_List_Area>
  <ctli:Type_List>
    <ctli:type>Imager</ctli:type>
    <ctli:subtype>wide-field</ctli:subtype>
    <ctli:type>Spectrometer</ctli:type>
    <ctli:subtype>spectrograph</ctli:subtype>
    <ctli:subtype>multi-object</ctli:subtype>
  </ctli:Type_List>
</Type_List_Area>
```
Method 3:
```
<Type_List_Area>
    <ctli:type>Imager</ctli:type>
    <ctli:subtype>wide-field</ctli:subtype>
    <ctli:type>Spectrometer</ctli:type>
    <ctli:subtype>spectrograph</ctli:subtype>
    <ctli:subtype>multi-object</ctli:subtype>
</Type_List_Area>
```
Method 4:
```
<Type_List_Area>
  <ctli:Type_List>
    <ctli:type>Imager</ctli:type>
    <ctli:subtype>wide-field</ctli:subtype>
  </ctli:Type_List>
</Type_List_Area>
<Type_List_Area>
  <ctli:Type_List>
    <ctli:type>Spectrometer</ctli:type>
    <ctli:subtype>spectrograph</ctli:subtype>   
    <ctli:subtype>multi-object</ctli:subtype>
  </ctli:Type_List>
</Type_List_Area>
```

**Describe the solution you'd like**
I need to be able to describe both types and subtypes of each of the modes of multi-mode instruments, and have the subtypes properly associated with their respective major types.

**Describe alternatives you've considered**
See above. Context objects are required and instrument type attributes are potential search criteria. There is no alternative to the context requirement. Not supporting search for multi-mode instruments is not an acceptable alternative.

**LDD Dictionary Version**
1.16.0.0 and all preceding

**PDS4 IM Version**
ASAP. This will impact the ability to ingest essential ground-characterization data included in the DART mission.

**Need-by Date**
ASAP, in terms of build cycles. Data from this instrument are expected in early 2022, and would likely be registered in the March-June quarter.

**Additional context**
This capability will also be needed for migration of data from multi-mode instruments on Rosetta, Deep Impact, etc. That timeline is more flexible than that of the DART mission.

**Comment by lylehuber** [comment]
Yes, we thought that Method 2 was what we were allowing but it turned out not to be. Method 1 is going to be the option going forward. Posting new Ingest file shortly.

**Comment by acraugh** [comment]
That'll work. Thanks @lylehuber!
